<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Asistente Chat OpenAI</string>
     </attribute>
     <widget class="QTextEdit" name="Qtext_Mensaje">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Enviar_Mensaje">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Conversacion</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="Qtable_Historial_Conversacion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>511</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Dall E-2</string>
     </attribute>
     <widget class="QTextEdit" name="Qtext_Descripcion_Imagen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Generar_Imagen">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar Imagen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Imagen_OpenAI">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagen Open IA</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
